.github/workflows/linter.yml

```
@@ -0,0 +1,37 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  prettier:
+    name: Prettier
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - name: Cloning the repository
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+        uses: actions/checkout@v2
+      - name: Prettier Action
+        uses: creyD/prettier_action@v2.2
```

SHould have options, see https://github.com/marketplace/actions/prettier-action

I'd say set the following:
```suggestion
        uses: creyD/prettier_action@v2.2
        with:
            prettier_options: --check ".github/**/*.yml" "./*.{yml,md}"
```

Or something like that